script/musicPlayer.js

```diff
@@ -75,42 +81,55 @@ export function musicPlayerInit() {
 		currValue = valueVolume;
 		audioPlayer.volume = valueToVolume(valueVolume);
 		// console.log(`input: ${valueVolume}, volume: ${audioPlayer.volume}`);
+
+		toggleVolumeIcon(audioVolume, audioMute);
 	};
 
+
 	const muteVolume = () => {
 		let prevValue = audioVolume.value;
 		audioPlayer.muted = !audioPlayer.muted;
 		if (prevValue != 0) {
+			currValue = audio.volume;
 			audioVolume.value = 0;
 			currValue = prevValue;
+			audioMute.classList.remove('fa-volume-down');
+			audioMute.classList.add('fa-volume-off');
 		} else {
 			audioVolume.value = currValue;
+			audioMute.classList.add('fa-volume-down');
+			audioMute.classList.remove('fa-volume-off');
 		}
-	}
+	};
 
+	// const maxVolume = () => {
+	// 	const maxValue = 100;
+	// 	let prevValue = audioVolume.value;
+	// 	if (audioPlayer.muted) {
+	// 		audioPlayer.muted = false;
+	// 	}
+	// 	if (prevValue != maxValue) {
+	// 		audioVolume.value = maxValue;
+	// 		currValue = prevValue;
+	// 	} else {
+	// 		audioVolume.value = currValue;
+	// 	}
 
-	const maxVolume = () => {
-		const maxValue = 100;
-		let prevValue = audioVolume.value;
-		if (audioPlayer.muted) {
-			audioPlayer.muted = false;
-		}
-		if (prevValue != maxValue) {
-			audioVolume.value = maxValue;
-			currValue = prevValue;
-		} else {
-			audioVolume.value = currValue;
-		}
+	// 	audioPlayer.volume = valueToVolume(audioVolume.value);
 
-		audioPlayer.volume = valueToVolume(audioVolume.value);
-	};
+	// 	console.log(audioPlayer.volume);
+	// };
+
+	maxVolume(audioVolume, audioPlayer);
+
+
+
+	// !!!! ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Volume ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 	audioMute.addEventListener('click', muteVolume);
 	volumeUp.addEventListener('click', maxVolume);
```

Comment:
here we need to call our exported maxVolume function:
here you could use 2 options:
1) use .bind() for correct adding of event listener 
2) create new function that returns exported maxVolume and pass it to event listener.